src/managers/input/InputManager.cpp

```diff
@@ -167,12 +167,16 @@ void CInputManager::mouseMoveUnified(uint32_t time, bool refocus) {
     Vector2D surfaceLocal = surfacePos == Vector2D(-1337, -1337) ? surfaceCoords : mouseCoords - surfacePos;
 
     if (pFoundWindow) {
-        if (g_pConfigManager->getInt("input:follow_mouse") == 0 && !refocus) {
+        const int FOLLOWMOUSE = g_pConfigManager->getInt("input:follow_mouse");
```

Comment:
can you make it speedier by making it

```
static auto *const PFOLLOWMOUSE = &g_pConfigManager->getrawptr(name)->intvalue;

if (*PFOLLOWMOUSE)

...
```

please?

Reply:
Done and tested 👍